docs/admin/install/install.rst

```diff
@@ -1,110 +1,14 @@
-
-In order to set up KeePassXC for easy use:
-
-- Add KeePassXC to the application menu by selecting it from the list of available apps in **Q > Apps > vault > Settings > Applications** and pressing the button labeled **>** (do not press the button labeled **>>**, which will add *all* applications to the menu).
-
-- Launch KeePassXC via **Q > Apps > vault > KeePassXC**. When prompted to enable automatic updates, decline. ``vault`` is networkless, so the built-in update check will fail; the app will be updated through system updates instead.
-
-- Close the application.
-
-.. important::
-
-   The *Admin Workstation* password database contains sensitive credentials not required by journalist users. Make sure to copy the credentials from the *Journalist Workstation* USB.
-
-In order to copy a journalist's login credentials:
-
-- If a *Journalist Workstation* USB is not currently attached, connect it, attach it to the ``vault`` VM, open it in the file manager, and enter its encryption passphrase.
-
-- Locate the password database. It should be in the ``Persistent`` directory, and will typically be named ``keepassx.kdbx`` or similar.
-
-- Open a second ``vault`` file manager window (``Ctrl + N`` in the current window) and navigate to the **Home** directory.
-
-- Drag and drop the password database to copy it.
-
-- In the ``vault`` file manager, right-click on the **TailsData** sidebar entry, then select **Unmount** and disconnect the *Journalist Workstation* USB. Close this file manager window.
-
-- In the file manager window that displays the home directory, open the copy you made of the password database by double-clicking it.
-
-- If the database is passwordless, KeePassXC may display a security warning when opening it. To preserve convenient passwordless access, you can protect the database using a key file, via **Database > Database settings > Security > Add additional protection > Add Key File > Generate**. This key file has to be selected when you open the database, but KeePassXC will remember the last selection.
-
-- Inspect each section of the password database to ensure that it contains only the information required by the journalist user to log in.
-
-- Close the application window and shut down the ``vault`` VM (using the Qube widget in the upper right panel).
-
 .. _download_rpm:
 
-Download and install SecureDrop Workstation
-~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+Download SecureDrop Workstation Packages
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-With the key and configuration available in ``dom0``, you're ready to set up SecureDrop Workstation:
+First, we'll download the SecureDrop Workstation packages needed to configure and install the SecureDrop Workstation.
```

Comment:
```suggestion
First, you must download the packages needed to configure and install the SecureDrop Workstation.
```
The way we address users changes throughout this PR. In some cases, we address them by saying that "we'll download/configure/etc.", but in other places we say "you will need to download/configure/etc."

We should aim to be be consistent - I recommend that we stick to "you must / you should, etc." just for consistency across all our docs.